lib/vmdb/productization.rb

```diff
@@ -24,7 +24,7 @@ def prepare_asset_paths
     #   The original value is located at:
     #     https://github.com/rails/rails/blob/v3.2.15/railties/lib/rails/application/configuration.rb#L48-L49
     def prepare_asset_precompilation
-      Rails.application.config.assets.precompile = [
+      Rails.application.config.assets.precompile += [
```

Comment:
I think the proc is there to include non `js`/`css` assets. Is that still needed?
I can add productization.css / js to the main array. wonder if that will blow up if that file is not available